test/integration/jobset_controller_test.go

```diff
@@ -311,6 +308,87 @@ var _ = ginkgo.Describe("JobSet controller", func() {
 			gomega.Eventually(checkJobSetStatus, timeout, interval).WithArguments(js, jobset.JobSetFailed).Should(gomega.Equal(true))
 		})
 	})
+	ginkgo.When("a jobset is created with suspend disabled", func() {
+		ginkgo.It("should create all jobs and headless services, then complete successfully once all jobs are completed", func() {
```

Comment:
this test is not needed, it is similar to one of the tests above.

Reply:
done.